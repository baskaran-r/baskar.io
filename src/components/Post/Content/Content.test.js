// @flow strict
import React from 'react';
import renderer from 'react-test-renderer';
import Content from './Content';

describe('Content', () => {
  it('renders correctly', () => {
    const props = {
      title: 'test',
      body: '<p>test</p>',
      date: '2020-12-22T03:30:13.175Z'
    };

    const tree = renderer.create(<Content {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
