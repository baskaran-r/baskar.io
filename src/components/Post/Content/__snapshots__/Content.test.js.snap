// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders correctly 1`] = `
<div
  className="content"
>
  <h1
    className="content__title"
  >
    test
  </h1>
  <div
    className="content__highlight-first-char"
  >
    T
  </div>
  <div
    className="meta"
  >
    <p
      className="meta__date"
    >
      Dec 22, 2020
    </p>
  </div>
  <div
    className="content__body"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<p>test</p>",
      }
    }
  />
</div>
`;
